src/itemsList/__snapshots__/ItemsList.unit.tsx.snap

```diff
@@ -796,6 +796,326 @@ button:focus:not(.focus-visible).c1 {
 </ul>
 `;
 
+exports[`ItemsList Should render an unordered list with only one element 1`] = `
```

Comment:
nit: If it's not too long, you might want to migrate this to RTL.

Reply:
Done